### PR TITLE
Update_on_provider_change

### PR DESCRIPTION
**Pull request by NickGonzalez04**
- To change the provider being set with a defined 'provider' variable. Didn't explain why it needed to be a variable after updating our code.

### CODE REVIEW

File: Solana_And_Web3/en/Section_2/Lesson_3_Store_Basic_Data.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ const { SystemProgram } = anchor.web3;
 const main = async() => {
   console.log("🚀 Starting test...")
 
+	// Create and set the provider. We set it before but we needed to update it, so that it can communicate with our frontend!
```
Comment (adilanchian) [inline Solana_And_Web3/en/Section_2/Lesson_3_Store_Basic_Data.md:133]: It looks like the indentation on this is a tad off!

Can you fix for me please :)?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Good catch!

Left one small comment here :). Thanks!

File: Solana_And_Web3/en/Section_2/Lesson_3_Store_Basic_Data.md (hunk 1 above)
Comment (NickGonzalez04) [reply Solana_And_Web3/en/Section_2/Lesson_3_Store_Basic_Data.md]: Done! :0

**Comment by adilanchian** [comment]
POGGG thank you :)

**Pull request merged**